product_state/views/product_views.xml

```diff
@@ -1,29 +1,149 @@
+                <field name="description"/>
+                <templates>
+                    <t t-name="kanban-box">
+                        <div class="oe_kanban_global_click">
+                            <div class="oe_kanban_details">
+                                <h4>
+                                    <field name="name"/>
+                                </h4>
+                                <div>
+                                    <a name="%(product_state.action_open_state_products)d" type="action">
+                                        <t t-esc="record.products_count.value"/> Products
+                                    </a>
+                                </div>
+                            </div>
+                        </div>
+                    </t>
+                </templates>
+            </kanban>
+        </field>
+    </record>
+
+    <record id="view_product_template_search_state" model="ir.ui.view">
+        <field name="name">product.template.search.state</field>
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="product.product_template_search_view"/>
+        <field name="arch" type="xml">
+            <field name="name" position="after">
+                <field name="product_state"/>
+                <filter string="State" name="groupby_state" domain="[]"
+                        context="{'group_by' : 'product_state'}"/>
+                <separator/>
+            </field>
+        </field>
+    </record>
 
     <record id="product_template_form_view" model="ir.ui.view">
         <field name="name">product.template.common.form</field>
         <field name="model">product.template</field>
         <field name="inherit_id" ref="product.product_template_form_view"/>
         <field name="arch" type="xml">
-            <sheet position="before">
-                <header>
-                    <field name="state" widget="statusbar" clickable="True"/>
-                </header>
-            </sheet>
+            <header position="inside">
+                <field name="product_state" widget="statusbar" clickable="True"/>
+                <field name="state" invisible="1"/>
+            </header>
         </field>
     </record>
 
-    <record id="product_template_view_tree_inherit_product_state" model="ir.ui.view">
-        <field name="name">product.template.tree (product_state)</field>
+    <record id="product_template_view_tree_inherit_product_state_id" model="ir.ui.view">
+        <field name="name">product.template.tree (product_state_id)</field>
         <field name="model">product.template</field>
         <field name="inherit_id" ref="product.product_template_tree_view"/>
         <field name="priority">50</field>
         <field name="arch" type="xml">
             <field name="type" position="after">
-                <field name="state"/>
+                <field name="product_state"/>
             </field>
         </field>
     </record>
 
```

Comment:
the same?

Reply:
I thought it was just in the product_template.py file